tools/fusion_standardization.cwl

```diff
@@ -0,0 +1,35 @@
+cwlVersion: v1.2
+class: CommandLineTool
+id: fusion-standardization
+requirements:
+  - class: ShellCommandRequirement
+  - class: InlineJavascriptRequirement
+  - class: ResourceRequirement
+    ramMin: 4000
+    coresMin: 2
+  - class: DockerRequirement
+    dockerPull: 'pgc-images.sbgenomics.com/d3b-bixu/annofuse:0.91.0'
+  - class: InitialWorkDirRequirement
+    listing:
+      - entryname: fusion_standardization_wrapper.R
+        entry:
+          $include: ../scripts/fusion_standardization_wrapper.R
+
+baseCommand: [Rscript, fusion_standardization_wrapper.R]
+
+inputs:
+  fusions_tsv: { type: File, doc: "Input fusion TSV file, i.e. fusion-dgd.tsv.gz. Can be compressed or uncompressed",
+    inputBinding: {position: 1, prefix: "--fusions_tsv"} }
+  caller: { type: [{type: enum, name: caller, symbols: ["STARFUSION", "ARRIBA", "DGD", "CUSTOM"]}], doc: "Caller used to produce input",
```

Comment:
```suggestion
  caller: { type: [{type: enum, name: caller, symbols: ["STARFUSION", "ARRIBA", "DGD", "CUSTOM"]}], doc: "Caller used to produce input",
```

aren't DGD and custom one in the same?

Reply:
Som they should be, but I Think Saksham, when he made this, was expecting the input to also have Gene2A and and Gene2B and annots already. I had thought that the behavior of adding the config file would be that it'd add those dummy columns if missing, instead he wrote them all as required. That's why, if you look at the R script, I pre-fill the DGD dataframe before passing to the function. We can either wait to fix that behavior now or later.